### DISCUSSION TITLE
Handled vs Unhandled Exceptions.

### DISCUSSION THREAD

**Discussion by iBotPeaches** [getsentry/sentry-react-native#2553] [Q&A] [answered]
Our team engulfed an older application (React Native) that was running RN Sentry 2.6.0, so we went through the upgrade the past few weeks and took up to the latest.

One thing that immediately changed for us was a huge influx of errors once deployed to production. On further investigation though this was "Handled" exceptions vs "Unhandled". So all the new errors we saw were handled exceptions that were properly handled and not much of a concern. I'm guessing the old Sentry 2.6.0 didn't upstream "handled" exceptions.

However, to the client this was quite a scare and saw thousands of new errors. I started doing some research and found this PR on the Java side: https://github.com/getsentry/sentry-java/pull/1248, which seemed to change the default to NOT send handled exceptions.

I'm just hoping to get some clarity or even configurations that would allow me to change the behavior when upstreaming handled exceptions to Sentry. For new projects and such, I'd keep it as-is and unchanged but engulfing such a large application with thousands of exceptions that are handled - I would like to somehow disable this and learn a bit about this distinction.

**Comment by marandaneto** [comment, marked as answer]
@iBotPeaches https://github.com/getsentry/sentry-java/pull/1248 is related to Spring only, so not related to this at all.
I don't understand what you mean by:

> 'm guessing the old Sentry 2.6.0 didn't upstream "handled" exceptions.

Do you mean that some events were not supposed to be sent to Sentry?
Or do you mean that events were not grouped together anymore and raised new issues that were already fixed in the past?
Please clarify your issue and give us more context.
Also, in case you have sentry.io issue links for us, so we can look into the events, thanks.

**Reply by iBotPeaches** [reply]
Apologies @marandaneto. My questions don't make much sense after further research. I was under false impression that older Sentry versions did not upload "Handled" errors to Sentry. My guess is Sentry got better at capturing errors between 2.6.0 and 4.6.1 and led to an influx of previously untracked issues.
